ci/docker/runtime_functions.sh

```diff
@@ -512,6 +519,7 @@ build_ubuntu_gpu_clang10_werror() {
        -DMXNET_CUDA_ARCH="$CI_CMAKE_CUDA_ARCH" \
        -DCMAKE_BUILD_TYPE="RelWithDebInfo" \
        -DUSE_CPP_PACKAGE=OFF \
+       -DUSE_INT64_TENSOR_SIZE=OFF \
```

Comment:
Why set it to OFF here?

Reply:
disabling on miscellaneous Clang build since they fail even with the cmake flag `MKL_USE_ILP64`
I followed the instructions here(https://software.intel.com/content/www/us/en/develop/documentation/mkl-macos-developer-guide/top/linking-your-application-with-the-intel-math-kernel-library/linking-in-detail/linking-with-interface-libraries/using-the-ilp64-interface-vs-lp64-interface.html) as suggested by @TaoLv. I works for Unix but not on CLang. I intend to fix it in stage 2

Reply:
What's the reason for splitting this up into multiple stages? The current PR is only changing the default settings (enable the setting by default). As long as there are bugs in various environments (eg with clang), you will break users. Thus I recommend including all required fixes in this PR. Also note that this clang build here has nothing to do with `MKL_USE_ILP64`